tgui/packages/tgui/interfaces/Radio/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useBackend } from "../../backend";
+import {
+  AnimatedNumber,
+  Box,
+  Button,
+  Knob,
+  LabeledList,
+  Section,
+  Stack,
+  Table,
+} from "../../components";
+import { formatFrequency } from "../../format";
+import { Window } from "../../layouts";
+import { RadioData, RadioWires } from "./type";
+
+export const Radio = (_props, context) => {
+  const { data, act } = useBackend<RadioData>(context);
+
+  const setFrequency = (value: number, finish: boolean) => {
+    act("set-frequency", { value, finish });
+  };
+
+  return (
+    <Window width="280" height="400" title={data.name}>
+      <Window.Content>
+        <Stack vertical fill>
+          <Stack.Item>
+            <Section>
+              <LabeledList>
+                <LabeledList.Item label="Microphone">
+                  <Button.Checkbox
+                    checked={data.broadcasting}
+                    onClick={() => act("toggle-broadcasting")}
+                  >
+                    {data.broadcasting ? "Engaged" : "Disengaged"}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Speaker">
+                  <Button.Checkbox
+                    checked={data.listening}
+                    onClick={() => act("toggle-listening")}
+                  >
+                    {data.listening ? "Engaged" : "Disengaged"}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Frequency">
+                  <Stack align="center">
+                    <Stack.Item>
+                      {!data.lockedFrequency && (
+                        <Knob
+                          animated
+                          value={data.frequency}
+                          minValue={1441}
+                          maxValue={1489}
```

Comment:
Throw these as constants at the top of the file, `MIN_FREQUENCY` and `MAX_FREQUENCY` or something.